aptos-move/framework/aptos-framework/sources/resource_account.move

```diff
@@ -248,7 +248,6 @@ module aptos_framework::resource_account {
     }
 
     #[test(framework = @0x1, user = @0x2345)]
-    #[expected_failure(abort_code = 0x60005, location = aptos_framework::coin)]
```

Comment:
Can we name the test to better document what the test actually tests for? Otherwise this seems like a obscure removal and test modification